src/core/bitop.d

```diff
@@ -276,15 +291,21 @@ version (AnyX86)
  * They may be used to guarantee a write or read cycle occurs at a specified address.
  */
 
-ubyte  volatileLoad(ubyte * ptr);
-ushort volatileLoad(ushort* ptr);  /// ditto
-uint   volatileLoad(uint  * ptr);  /// ditto
-ulong  volatileLoad(ulong * ptr);  /// ditto
+pragma(DigitalMars_intrinsic, "ind")
```

Comment:
These are the dmd backend op names.  Do you have any ideas on a good namespacing scheme for builtins?  Does it make sense to have some common ones?

Something like `digitalmars.*` `gcc.*` `llvm.*` `d.*`?